glustolibs-gluster/glustolibs/gluster/lib_utils.py

```diff
@@ -697,6 +697,14 @@ def install_epel(servers):
                 if ret != 0:
                     g.log.error("Epel install failed")
                     rt = False
+            elif (("release 8" in release_string) or
```

Comment:
@srivickynesh Just to understand, lets say if its 'Fedora' in the release string, the control goes to the elif condition above (https://github.com/gluster/glusto-tests/blob/d147a419e5c99484a51cfcb956356748c45d76ab/glustolibs-gluster/glustolibs/gluster/lib_utils.py#L693) and not this elif statement. Is that acceptable or do we have to change the logic here?